admin_guide/persistent_storage/pod_security_context.adoc

```diff
@@ -0,0 +1,243 @@
+= Pod Security Context: Volume security
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+This document provides a general guide on pod security context as it
+relates to volume security. For information on pod security context in
+general please refer to link:../manage_scc.html[Managing Security
+Context Constrains] documentation.
+
+The Pod Security Context provides three fields which allow you to gain
+and control access to volumes. These are
+`FSGroup`,`SupplementalGroups`, and `SELinuxOptions`.
+
+[IMPORTANT]
+====
+It is important to note that `FSGroup` and `SELinuxOptions` are
+generated for your pod if not specified. It is recommended that you
+rely on the automatically generated values and not specify your own
+unless you have a specific use case which requires that.
+====
+
+== SupplementalGroups
+
+Supplemental groups are regular Linux groups. When a process run in
+Linux it has UID and one or more GID's. Usually this is just the UID,
+and GID's of user who started the process, but in the case of
+containers you have the option of settings these attributes for the
+container's main process. The `SupplementalGroups` option of the pod
+`SecurityContext` takes advantage of that and enables you to specify a
+list of groups which will be added to the main process of every
+container in the pod.
+
+This is useful because it enables you to give group based access to
+volumes to your pods.
+
+Let's take a look at an example.
+
+Say you have an NFS export as follows:
+
+====
+----
+showmount -e example.com
+Export list for example.com:
+/nfsexport (everyone)
+
+# ls -ld /nfsexport # on NFS server
+drwxrwx---. 2 root 1234 4096 Oct 30 15:27 /nfsexport
+----
+====
+
+The above export is only accessible root and the group 1234. So any
+pod which is not run as root and is not a member the group 1234 will
+not be able to access that export. It is often the case that the in
+the container cannot be controlled or predicted.
+
+To solve this issue we add the group 1234 SupplementalGroups as in the
+following example:
+
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nfs-web
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+      volumeMounts:
+          - name: nfs
+            mountPath: "/usr/share/nginx/html"
+  securityContext:
+    supplementalGroups: [1234]
+  volumes:
+    - name: nfs
+      nfs:
+        server: example.com
+        path: "/nfsexport"
+----
+
+====
+
+All containers in the above pod will be members of the group 1234 and
+will have access to the volume regardless of what the container
+runs as.
+
+This is useful with volumes which do not allow you to change and
```

Comment:
"to change and ownership" ? Maybe "to change any ownership" or just "to change ownership"?